media/utils/addMostRelevantJiraTicket.js

```diff
@@ -46,7 +46,10 @@ const addMostRelevantJiraTicket = (jiraTickets) => {
     if (jiraTickets.length > 1) {
       addViewAlTicketssButton(jiraTickets);
     }
-  }
+  } else {
+    $("#mostRelevantJiraTicketHolder").append(
+      `<p>Jira is a premium feature, <a href="https://www.watermelontools.com/pages/pricing">click here to buy</a></p>`
```

Comment:
It's ok, but text could be something better. Something that shows the benefit instead of asking for money so explicitly. For example: "Click here to see the most relevant Jira ticket for this block of code", or "Integrate Jira to your code archeology toolbox".

Reply:
I have merged the three phrases.